csharp/ql/test/resources/stubs/Antlr3.Runtime/3.5.1/Antlr3.Runtime.cs

```diff
@@ -1,46 +1,48 @@
-            public ANTLRInputStream(System.IO.Stream input, int size, System.Text.Encoding encoding) : base(default(System.IO.TextReader)) => throw null;
+            public ANTLRInputStream(System.IO.Stream input) : base(default(System.IO.TextReader)) => throw null;
             public ANTLRInputStream(System.IO.Stream input, int size) : base(default(System.IO.TextReader)) => throw null;
             public ANTLRInputStream(System.IO.Stream input, System.Text.Encoding encoding) : base(default(System.IO.TextReader)) => throw null;
-            public ANTLRInputStream(System.IO.Stream input) : base(default(System.IO.TextReader)) => throw null;
+            public ANTLRInputStream(System.IO.Stream input, int size, System.Text.Encoding encoding) : base(default(System.IO.TextReader)) => throw null;
+            public ANTLRInputStream(System.IO.Stream input, int size, int readBufferSize, System.Text.Encoding encoding) : base(default(System.IO.TextReader)) => throw null;
         }
-
-        // Generated from `Antlr.Runtime.ANTLRReaderStream` in `Antlr3.Runtime, Version=3.5.0.2, Culture=neutral, PublicKeyToken=eb42632606e9261f`
         public class ANTLRReaderStream : Antlr.Runtime.ANTLRStringStream
         {
-            public ANTLRReaderStream(System.IO.TextReader r, int size, int readChunkSize) => throw null;
-            public ANTLRReaderStream(System.IO.TextReader r, int size) => throw null;
             public ANTLRReaderStream(System.IO.TextReader r) => throw null;
-            public const int InitialBufferSize = default;
+            public ANTLRReaderStream(System.IO.TextReader r, int size) => throw null;
+            public ANTLRReaderStream(System.IO.TextReader r, int size, int readChunkSize) => throw null;
+            public static int InitialBufferSize;
             public virtual void Load(System.IO.TextReader r, int size, int readChunkSize) => throw null;
-            public const int ReadBufferSize = default;
+            public static int ReadBufferSize;
         }
-
-        // Generated from `Antlr.Runtime.ANTLRStringStream` in `Antlr3.Runtime, Version=3.5.0.2, Culture=neutral, PublicKeyToken=eb42632606e9261f`
-        public class ANTLRStringStream : Antlr.Runtime.IIntStream, Antlr.Runtime.ICharStream
+        public class ANTLRStringStream : Antlr.Runtime.ICharStream, Antlr.Runtime.IIntStream
         {
-            public ANTLRStringStream(string input, string sourceName) => throw null;
-            public ANTLRStringStream(string input) => throw null;
-            public ANTLRStringStream(System.Char[] data, int numberOfActualCharsInArray, string sourceName) => throw null;
-            public ANTLRStringStream(System.Char[] data, int numberOfActualCharsInArray) => throw null;
-            protected ANTLRStringStream() => throw null;
-            public virtual int CharPositionInLine { get => throw null; set => throw null; }
+            public virtual int CharPositionInLine { get => throw null; set { } }
```

Comment:
`set` doesn't throw null?

Reply:
No reason to, it is just a matter of making the stubs as simple as possible in terms of generated code.